### DISCUSSION TITLE
reduce size and wrap the issue labels on android

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3682] [Mobile] [answered]
this just generally looks better on ios

ios
| 1 | 2 |
| - | - |
| ![ios_1](https://user-images.githubusercontent.com/39104740/113486945-dd738f80-94bd-11eb-84d3-32766d6acb88.PNG) | ![ios_2](https://user-images.githubusercontent.com/39104740/113486948-dfd5e980-94bd-11eb-9aeb-4ec148e02742.PNG) |

android

<img width="300" alt="android" src="https://user-images.githubusercontent.com/39104740/113486956-f11ef600-94bd-11eb-8557-2a4675f3d6d5.jpg">

**Comment by nt4f04uNd** [comment]
also, i noticed on these screenshots that on ios when you go to issue/pull reqests/etc from the main screen, the navigation rail is intact so you can easier navigate, this would be nice to see on android as well

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback @nt4f04uNd – agreed on the labels looking incorrect and not wrapping on Android, this is something we're working on right now and will fix soon 😊
